### PR TITLE
Issue #6 - Turn "previous events" into a timeline

### PR DESCRIPTION
**Pull request by jvzammit**
Wish I had retained markdown, but HTML markup is quite readable IMHO.

Please take a good look at SCSS as I'm really out of my depth on that one, thank you!

### CODE REVIEW

File: assets/sass/bootstrap/_timeline.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
+//
+// Base styles
+//
+
+ul.timeline {
+	list-style-type: none;
+	position: relative;
+	&:before {
+		content: ' ';
+		background: #d4d9df;
```
Comment (benjaoming) [inline assets/sass/bootstrap/_timeline.scss]: You can use one of the theme's color variables here.

```suggestion
		background: $info;
```

File: assets/sass/bootstrap/_timeline.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	>li {
+		margin: 20px 0;
+		padding-left: 20px;
+		&:before {
+			content: ' ';
+			background: white;
+			display: inline-block;
+			position: absolute;
+			border-radius: 50%;
+			border: 3px solid rgb(218, 65, 103);
```
Comment (benjaoming) [inline assets/sass/bootstrap/_timeline.scss]:
```suggestion
			border: 3px solid $info;
```

**Review by benjaoming — APPROVED** [review]
This works so well! And looks good! Thanks @jvzammit - I think your first SCSS is really nice!

I moved it out of the "vanilla" Bootstrap so it can be easy in the future to do updates of Bootstrap and keep our custom stuff outside.

File: assets/sass/djangoday.scss [hunk 3]
```diff
@@ -298,3 +298,36 @@ body.some {
...
+			margin-top: 3px;
+			border-radius: 50%;
+			border: 3px solid $primary;
+			left: 20px;
+			width: 20px;
+			height: 20px;
+			z-index: 400;
+		}
+	}
+}
```
Comment (jvzammit) [inline assets/sass/djangoday.scss:333]: 👍 good move!

**Pull request merged**